tracardi/process_engine/action/v1/converters/base64/encode/model/models.py

```diff
@@ -0,0 +1,6 @@
+from tracardi.service.plugin.domain.config import PluginConfig
+
+
+class Configuration(PluginConfig):
+    to_base64: str
```

Comment:
I'm unsure about the name for this property. I guess it's okay, but I haven't found any documentation or pattern of the existing configurations. Is there any preference?

Perhaps simply `input`, `path`, `input_path`, `property`, `property_path`, or `dot_path`?

Reply:
Okay, will go with `source`; I think that describes it pretty well